lib/index.coffee

```diff
@@ -20,9 +22,27 @@ module.exports =
       loadModule()
       packageSync.sync()
 
+    atom.packages.onDidActivateInitialPackages ->
+      @didActivateInitialPackages = true
```

Comment:
Rather than storing a status variable, you could add the `onDidLoadPackage` and `onDidUnloadPackage` handlers inside the `onDidActivateInitialPackages` handler.